101-aks-azml-targetcompute/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
add the environments property to exclude Azure Gov support

Reply:
fixed, thanks